lib/index.js

```diff
@@ -8,6 +8,7 @@ var Spawn = require("child_process").spawn
   , Ul = require("ul")
   , OneByOne = require("one-by-one")
   , Sliced = require("sliced")
+  , Q = require('q')
```

Comment:
@IonicaBizau @RubenVerborgh I am still pretty fresh in nodejs ecosystem and literally yesterday I learned about libs like promisify - I think that this is how it should work :D 

Having only 'nodeback' API is more than okay then - one can easily promisfy it when needed. Sorry for confusion and thanks for review guys! :D I think that we can close this PR.